src/Jobs/Feeds/Feed.php

```diff
@@ -472,4 +469,36 @@ public function uniqueId()
     {
         return $this->vendor.$this->store->code;
     }
+
+    public function getProductsCount($query)
+    {
+        $collection = collect([]);
```

Comment:
What is this collection doing? It looks like it's doing nothing and just returns `$query->count()`